modules/web/js/ballerina/components/block-statement-decorator.jsx

```diff
@@ -63,22 +85,26 @@ class BlockStatementDecorator extends React.Component {
      * Navigates to codeline in the source view from the design view node
      *
      */
-    onJumptoCodeLine() {
-        const { renderingContext: { ballerinaFileEditor } } = this.context;
-
-        const container = ballerinaFileEditor._container;
-        $(container).find('.view-source-btn').trigger('click');
+    onJumpToCodeLine() {
+        document.getElementsByClassName('view-source-btn')[0].click();
     }
 
-    onUpdate(text) {
+    /**
+     * Call-back for when a new value is entered via expression editor.
+     */
+    onUpdate() {
```

Comment:
Ya it's not used yet but i though @kaviththiranga was planing to use it to validate later, I'll remove it for now then.